minigui/serve.py

```diff
@@ -43,6 +49,13 @@
          "\"cc\" for the C++ implementation.")
 
 
+parser.add_argument(
```

Comment:
why can't this just be the python on your PATH?

Reply:
If you're running in a virtualenv, sure, but if you're installing python inside of Ubuntu, then you need python2 as python for google-cloud-sdk and python3 has to then remain python3. And so needs to be passed into container-land.